tests/default/_core/search/geo_bounding_box.yaml

```diff
@@ -0,0 +1,61 @@
+$schema: ../../../../json_schemas/test_story.schema.yaml
```

Comment:
Split this into multiple tests and move under the `query` folder:

- _core/search/query/xy_point.yaml
- _core/search/query/xy_shape.yaml
- _core/search/query/geo_bounding_box.yaml

Reading https://github.com/opensearch-project/geospatial/blob/main/release-notes/opensearch-geospatial.release-notes-2.4.0.0.md I think this is only supposed to work post 2.4, or at least I'd start there, so add `version: >= 2.4` into the tests and it will skip older versions of OpenSearch.